infra/presubmit.py

```diff
@@ -18,12 +18,19 @@
 
 import argparse
 import os
+import re
 import subprocess
 import sys
 import unittest
 import yaml
 
 _SRC_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+TEST_BLOCKLIST = [
+    re.compile(r'.*\/infra\/base-images\/base-sanitizer-libs-builder'),
```

Comment:
Why is this one blacklisted ?

Reply:
I explain now in a comment that it's because of an error: `ModuleNotFoundError: No module named 'apt'`